docs/guides/installation/docker.md

```diff
@@ -59,7 +59,7 @@ services:
       - 6379:6379
 
   directus:
-    image: directus/directus:v9.0.0-rc.19
+    image: directus/directus:v9.0.0-rc.24
```

Comment:
Is there a way to make this "latest" instead of a specific version? Otherwise we'll have this same problem within 24h

Reply:
Not until we make a final release. Then we can change it to :v9 or :latest

Reply:
This was just to solve the cache hang issue until we the GA release.

Reply:
Aren't we pushing the RCs to `latest`? Why not?